centos-7.template

```diff
@@ -21,6 +21,7 @@ repo --name=base --baseurl=http://mirror.centos.org/centos/7/os/x86_64/
 repo --name=updates --baseurl=http://mirror.centos.org/centos/7/updates/x86_64/
 repo --name=extras --baseurl=http://mirror.centos.org/centos/7/extras/x86_64/
 repo --name=atomic --baseurl=http://mirror.centos.org/centos/7/atomic/x86_64/adb/
+repo --name=hvkvp --baseurl=http://files.gbraad.nl/hvkvp/
```

Comment:
I think COPR yum repo would be better choice as  you can give access to other members to team to maintain the RPM when required.